### ISSUE TITLE
When showing vorpal from inside a callback ctrl-c twice does not exit

### ISSUE THREAD

**Issue by sarus** [dthree/vorpal#70]
I'm running some async init code before I show vorpal and so ctrl-c twice no longer exits vorpal as I'm stuck in my callback.  The idea here is that I only want to run vorpal if I can successfully connect to a database since all the vorpal commands are db related.

```
// basic setup
testConnection(function(err, result){
   if(!err){
        vorpal.show();
   }
});
```

Is there an event or callback I can listen for to know when vorpal exists?  Right now hitting ctrl-c appears to be stopping vorpal but Node is sitting in my callback waiting for something.  I realize I might also have this setup wrong.

If I add the following:

```
process.on('SIGINT', () => process.exit(2));
```

Then I can get my program to quit if I hti ctrl-c three times.  Otherwise I have to `kill -9` it.

Thanks!

**Comment by dthree** [comment]
Dude you confused the hell out of my with your avatar...

**Comment by eliperelman** [comment]
Haha nice.

**Comment by dthree** [comment]
Related:

https://github.com/dthree/vorpal/issues/55

**Comment by sarus** [comment]
@dthree haha.  I was super confused when I first came across this repo as well.

**Comment by dthree** [comment]
Haha - as you do. Even our positioning is the same...

**Comment by dthree** [comment]
Fixed!! `v1.5.2`.

See https://github.com/dthree/vorpal/issues/55#issuecomment-167161798
